vendor/assets/javascripts/dataTables.bootstrap.js

```diff
@@ -0,0 +1,186 @@
+/*! DataTables Bootstrap 3 integration
```

Comment:
do we need this file? I was hoping we could keep js / css like that out of the repo by using https://github.com/rweng/jquery-datatables-rails, which I think has a bootstrap3 flavour, but perhaps there are some customisations involved?